modules/compiler/source/base/source/image_argument_substitution_pass.inc

```diff
@@ -120,10 +120,8 @@
 // intanium mangling mandates that built-ins are not substituted.
 // However,
 // clang 3.6 got it wrong and does do the substitution. By adding the
-// extra mapping we make sure that if clang ever gets fixed (or we are
-// consuming a binary generated by SPIR 1.2 generator) we can correctly
-// match the function. There is a funny twist to that issue, CTS SPIR
-// test
+// extra mapping we make sure that if clang ever gets fixed we can correctly
+ // match the function. There is a funny twist to that issue, CTS SPIR test
```

Comment:
weird indentation - also should we be talking about SPIR tests if we are skipping them?